core/store/ledgerstore/ledger_store.go

```diff
@@ -630,11 +629,19 @@ func (this *LedgerStoreImp) executeBlock(block *types.Block) (result store.Execu
 			return
 		}
 	}
+	gasTable := make(map[string]uint64)
+	neovm.GAS_TABLE.Range(func(k, value interface{}) bool {
```

Comment:
why not change neovm.GAS_TABLE from sync.map to map with RWLock?

Reply:
from the implementation perspective, there is little difference between `sync.map` and `map with RWLock`. clone a local map is a cheap way to avoid locking before each access.